### PR TITLE
GAWB-1251: Option to keywordize AJAX response

### PR DESCRIPTION
**Pull request by MattPutnam**
Commit 1 does the dirty work, Commit 2 implements the change in some of the uglier places.

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by davidangb** [comment]
(needs a rebase now) ... the changes in this PR look fine to me. I haven't tried to check if they are comprehensive, if you've missed anything, or if there are any regressions - just gave it a once-over as TL.

File: src/cljs/org/broadinstitute/firecloud_ui/utils.cljs [hunk 1]
```diff
@@ -121,8 +122,10 @@
                                      :success? (<= 200 status-code 299)
                                      :status-text (.-statusText xhr)
                                      :raw-response (.-responseText xhr)
-                                     :get-parsed-response #(parse-json-string
-                                                            (.-responseText xhr))})))]
+                                     :get-parsed-response
+                                     (fn [& [keywordize-keys?]]
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/utils.cljs:126]: What about:

``` clojure
(partial parse-json-string (.-responseText xhr))
```

**Review by dmohs — APPROVED** [review]
LGTM 👍

File: src/cljs/org/broadinstitute/firecloud_ui/utils.cljs (hunk 1 above)
Comment (MattPutnam) [reply src/cljs/org/broadinstitute/firecloud_ui/utils.cljs:126]: parse-json-string defaults to false, this defaults to true

**Pull request merged**